core.json

```diff
@@ -1103,9 +1107,27 @@
               "format":"int"
             }
           }
+        },
+        {
+          "name": "Old time",
+          "description": "A block for storing the old start time when an event has been rescheduled",
+          "declares": {"type": "core/event-old-start"},
+          "data": {
+            "dateGranularity": {"enum":["date", "datetime"]},
+            "start": {"format":"RFC3339"},
+            "updated": {"format":"RFC3339"}
+          }
         }
       ],
       "links": [
+        {
+          "name": "Replaced by",
+          "description": "The event has been moved to another day, and has been replaced by another event document.",
+          "declares": {"type": "core/event", "rel": "moved_to"},
```

Comment:
The information have not moved, rather copied. I think `rel="replaced_by"` would be more suitable to express the relationship, in line with the name.

Reply:
Yeah, I was thinking that the event had moved, not the document. But using "replaced-by" makes sense, changing.